package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "webpack-intro",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "dev": "webpack serve"
```

Comment:
**OPTIONAL** - Please make sure your package.json file has the `run` script, I am making this optional because your `dev` script works too. Please fix this 🙏

Reply:
I will fix it. Thank you 🙏